.github/workflows/verify.yml

```diff
@@ -144,24 +164,47 @@ jobs:
         with:
           files: "package.json"
 
+      - name: Check for bun.lockb existence
+        id: bun_lock_check
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "bun.lockb"
+
+      - name: Check if Sorbet is in use
```

Comment:
Looks like this was accidental, bad rebase?

Reply:
Good catch 😬 I've rebased my commits to remove the merge bubble and correct this error!